docs/docs/api-reference/core/selector.md

```diff
@@ -164,13 +164,16 @@ import {selector, useRecoilValue} from 'recoil';
 const myQuery = selector({
   key: 'MyDBQuery',
   get: async () => {
-    return myDBQueryPromise();
+    const queryResults = await myDBQueryPromise();
+    // other dependent operations
```

Comment:
How about:
```jsx
get: async () => {
  const queryResults = await myDBQueryPromise();
  return queryResults.data;
},
```

Reply:
sure, in case if you agree to add more prectical content I could replace it to smt like this
```
get: async () => {
  const response= await fetch(getMyRequest());
  return await response.json();
},
```

let me know which way do you prefer. Once the improvement will be confirmed I will update the PR.

Reply:
haha, maybe `JSON.parse(response)`.  But, seems `getMyRequestUrl()` would really be actually doing the request, not just getting the URL, so `getMyRequestUrl()` doesn't sound right.

Reply:
`fetch(https://path/to/url.json')` or `fetch(getUrlToFetchJSON())`?

Reply:
sure, with the `fetch()`: `await fetch(getMyRequestUrl())`

Reply:
it's what I mentioned initially. Anyway I will apply my example, please will comment what do you suppose to replace (for now I misundersand your point)

Reply:
I just missed the `fetch()` part, sorry.  But, please update `response.json()` to `JSON.parse(response)` or `JSON.parse(response.date)`

Reply:
could you clarify how do you expect it will work with any real request (common application/json content-type)?
my check raises correct execption
![image](https://user-images.githubusercontent.com/15786916/87126993-d6ed1a00-c295-11ea-8916-55408930d28e.png)

and current version works as expected
![image](https://user-images.githubusercontent.com/15786916/87127039-ea988080-c295-11ea-8ac1-fe51135c5076.png)

Reply:
Ah, I see `fetch()` is a specific Web API that returns an object with `json()`.